### PR TITLE
Us037 deploy frontend to aws

### PR DESCRIPTION
**Pull request by DarraghHaughey-Kainos**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  workflow_dispatch:
```
Comment (glenn-hamilton) [inline .github/workflows/docker.yml]: Shouldn't need this

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
```
Comment (glenn-hamilton) [inline .github/workflows/docker.yml]: Remember to remove this before it's merged

File: package.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "start": "npx tsc && cp -R views dist && cp -R public dist && cp -R images dist && node dist/app.js",
     "start-win": "npx tsc && copy -R views dist && copy -R public dist && copy -R images dist && node dist/app.js",
+    "dev": "ts-node-dev app.ts",
```
Comment (glenn-hamilton) [inline package.json]: Is this used anywhere?

Comment (DarraghHaughey-Kainos) [reply package.json]: no i was just checking it out lol i'll remove it and its dependencies now

**Review by glenn-hamilton — APPROVED** [review]

**Review by steveh2707 — APPROVED** [review]
looks top notch

**Pull request merged**